plugins/LocalFileOutputDevice/LocalFileOutputDevice.py

```diff
@@ -140,6 +139,7 @@ def requestWrite(self, nodes, file_name = None, limit_mimetypes = None, file_han
                 raise OutputDeviceError.UserCanceledError()
 
         self.writeStarted.emit(self)
+        Application.getInstance().workspaceLoaded.emit(file_name)
```

Comment:
I don't think that this is correct. This means that if you load two STL files that the second STL changes the file name.

Reply:
Nope. Loading STL files triggers the workspaceLoaded only if the buildplate is completely empty. Otherwise they are not workspaces so this isn't trigger again.

Reply:
The trigger being called is not correct either.

Basically this means that if any file is saved to local disk, we're pretending that we're "opening a workspace". No workspace has been opened, so anything that listens to this signal would expect that a workspace has been opened. But instead, a file has been loaded. Completely different situations.